src/index.ts

```diff
@@ -1,3 +1,64 @@
+import { Client } from './networks/client';
+import { EthereumClient } from './networks/evm';
+import { TronClient } from './networks/tron'
+
+import HDWalletProvider from '@truffle/hdwallet-provider';
+import { HDKey } from '@scure/bip32';
+import { mnemonicToSeedSync } from '@scure/bip39';
+
+
 export { Client } from './networks/client';
 export { EthereumClient } from './networks/evm';
-export { PolkadotClient } from './networks/polkadot';
\ No newline at end of file
+export { TronClient } from './networks/tron'
+
+export enum SupportedNetwork {
+    EvmNetwork,
+    TronNetwork,
+}
+
+export function networkType(chainId: number): SupportedNetwork | undefined {
```

Comment:
shouldn't we move this map to config as well?

Reply:
Not sure actually. I believe the application (only `zkbob-console` use this library for now) should know nothing about the underlying network type. The network client emited by this library is an abstraction for blockchain world. This method returns just a network type for internal usage and makes sense just inside surrport-js library. I think it's makes sense to remove this method from export to avoid confusing here

Reply:
This method becomes internal in https://github.com/zkBob/zeropool-support-js/pull/9/commits/08ebbb4b6e66cc14a0988740e4d23a46958f6ed5

Reply:
The application may not know anything about the underlying structures, but still pass a config downstream. We're already doing it with main DApp. Idk but having small config pieces here and there (libzkbob-rs, zeropool-support-js) that contain some magic number feels somewaht wrong, since ideally you would try to avoid any changes to source code in case of adding new pools or more instances of the same networks (eg Goerli will be replaced by a new testnet Holesky soon), new networks of the same type etc